pod/perlfunc.pod

```diff
@@ -7866,7 +7866,6 @@ produces the output C<abc>, and this:
 produces the output C<a:bc>, and each of these:
```

Comment:
Wouldn't really make sense for this (or the following text) to be plural with only one example